### PR TITLE
MCLOUD-5667: Add Universal Configuration documentation

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of this pull request

https://jira.corp.magento.com/browse/MCLOUD-5667

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-devbox.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+Or by running CLI command
+
+```bash
+composer require --dev "magento/magento-cloud-docker:^1.1"
+```
+
+## Configuration
+
+Magento Cloud Docker supports few configurations:
```
Comment (meker12) [inline src/cloud/docker/docker-devbox.md:34]:
```suggestion
Magento Cloud Docker supports the following configurations:
```

File: src/cloud/docker/docker-devbox.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    path: "app/etc"
+  pub-media:
+    path: "pub/media"
+  pub-static:
+    path: "pub/static"
+```
+
+## Fully custom configuration
+
+You can fully override the file configuration by using command `build:custom:compose`
```
Comment (meker12) [inline src/cloud/docker/docker-devbox.md:167]:
```suggestion
You can override the file configuration by using `build:custom:compose` command.
```

**Comment by meker12** [comment]
New content incorporated on branch [mae-mcloud-5667-docker-stand-alone-config](https://github.com/magento/devdocs/tree/mae-mcloud-5667-docker-stand-alone-config) - refactoring work for addition of universal Docker config file in progress.

**Pull request closed without merging**